File: 1014.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var distancia = Number(lines.shift());
var combustivel = Number(lines.shift());
var consumo = distancia / combustivel;
console.log(`${consumo.toFixed(3)} km/l`);

File: 1014.java
import java.util.Scanner;
import java.util.Locale;
public class Main {
public static void main(String[] args){
        Locale.setDefault(Locale.US);
        Scanner input = new Scanner(System.in);
        double distancia = input.nextDouble();
        double combustivel = input.nextDouble();
        double consumo = distancia / combustivel;
        System.out.printf("%.3f km/l%n", consumo);
        }
}
